src_assets/common/assets/web/index.html

```diff
@@ -109,6 +112,27 @@ <h3>{{githubVersion.name}}</h3>
       this.loading = false;
     },
     computed: {
+      installedVersionNotStable() {
+        if (!this.githubVersion || !this.version) {
+          return false;
+        }
+        let v_github = this.githubVersion.name;
+        if (v_github.indexOf("v") === 0) {
+          v_github = v_github.substring(1);
+        }
+        let v_this = this.version;
+        if (v_this.indexOf("v") === 0) {
+          v_this = v_this.substring(1);
+        }
+        let v_github_parts = v_github.split('.').map(Number);
+        let v_this_parts = v_this.split('.').map(Number);
+        for (let i = 0; i < Math.min(3, v_github_parts.length, v_this_parts.length); i++) {
+          if (v_this_parts[i] > v_github_parts[i]) {
+            return true;
+          }
+        }
```

Comment:
How about we have a `SunshineVersion` class, that receives this as constructor, and has the following props/methods

```
versionName
major
minor
patch
publishedAt

isStable(),
```

This way we use same parsing for both release/pre-release in the constructor, and just use this class instead, what do you think?